spec/cucumber/formatter/legacy_api/adapter_spec.rb

```diff
@@ -158,19 +158,19 @@ def after_hooks(source)
             end
           end
 
-          expect( formatter.legacy_messages ).to eq [
-              :before_features,
-                :before_feature,
+          expect(formatter.legacy_messages).to eq [
+            :before_features,
+              :before_feature,
+                :before_tags,
+                :after_tags,
+                :feature_name,
+                :before_feature_element,
                   :before_tags,
-                  :after_tags,
-                  :feature_name,
-                  :before_feature_element,
-                    :before_tags,
-                      :after_tags,
-                    :scenario_name,
-                  :after_feature_element,
-                :after_feature,
-              :after_features,
+                    :after_tags,
+                  :scenario_name,
+                :after_feature_element,
+              :after_feature,
+            :after_features,
           ]
```

Comment:
@nodo, the error that this PR fixes is related to this violation message:

```
spec/cucumber/formatter/legacy_api/adapter_spec.rb:162:15: C: Use 2 spaces for indentation in an array, relative to the start of the line where the left square bracket is.
              :before_features,
              ^^^^^^^^^^^^^^^^
```
So, I avoided doing any more changes other than the ones minimum needed to get rid of this violation 🙂  
Moving the indentation 2 spaces to the left (for the whole block) fixed that. Do you think that I should work on this with further enhancements? If yes, what enhancements would you suggest?